WalletWasabi.Gui/Global.cs

```diff
@@ -177,16 +177,16 @@ public async Task InitializeNoWalletAsync()
 
 				if (Config.UseTor)
 				{
-					TorManager = new TorProcessManager(Config.TorSocks5EndPoint, TorLogsFile);
+					TorManager = new TorProcessManager(Config.TorSocks5EndPoint, DataDir, TorLogsFile);
+					TorManager.Start(ensureRunning: false);
```

Comment:
Why was this moved here?

Reply:
It's mentioned as the last point in the PR description. 

Basically, the current design is that if you don't want to use Tor, then you set `torSocks5EndPoint` to `null` and `Start`, `StartMonitor`, `StopAsync` are then no-operation methods. This is not really useful when the alternative is simply not to use `TorProcessManager` when user does not want to use Tor. 

To elaborate a bit more, those Task.Delays in TorProcessManager makes me uneasy because it may happen that you want to use Tor and there is a chance that your first Tor request will fail because Tor is not loaded or some other very hard to debug case may happen. 

I believe that useful API is 

* `TorProcessManager.Start()` - start process but do not wait for it to be fully started
* `TorProcessManager.Start(ensureRunning: true)` - start process and make sure it is really running, then return.

What do you think?